internal/wallet/wallet.go

```diff
@@ -49,6 +50,10 @@ var (
 	waddrmgrNamespaceKey = []byte("waddrmgr")
 	waddrmgrKeyScope     = waddrmgr.KeyScopeBIP0084
 	wtxmgrNamespaceKey   = []byte("wtxmgr")
+
+	rpcport     = "localhost:18443" // TODO: have rpc params be read from conf file? //
+	rpcusername = "akek"
+	rpcpassword = "akek"
```

Comment:
Yes, the RPC parameter should be defiend in a conf file.
Leave the TODO and do it later when we integrate.